appveyor.yml

```diff
@@ -61,28 +62,32 @@ environment:
 init:
   - ps: echo $env:TESTENV
 install:
-  - set PATH=%PATH%;C:\msys64\mingw64\bin
+  # This quickly fails the job if there is another pipeline queued ahead of it
+  - ps: if ($env:APPVEYOR_PULL_REQUEST_NUMBER -and $env:APPVEYOR_BUILD_NUMBER -ne ((Invoke-RestMethod `
+          https://ci.appveyor.com/api/projects/$env:APPVEYOR_ACCOUNT_NAME/$env:APPVEYOR_PROJECT_SLUG/history?recordsNumber=50).builds | `
+          Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
+            throw "There are newer queued builds for this pull request, failing early." }
   - '%MINICONDA_HOME%\Scripts\conda config --set always_yes yes'
   - '%MINICONDA_HOME%\Scripts\conda update -q conda'
-  - '%MINICONDA_HOME%\Scripts\conda create -q -p C:\pythontest python=%PYTHON_VERSION% %INSTALL_LIBPYTHON% pytest numpy'
-  # remove file which conflicts with installation of package 'readme' on case-insensitive file systems
+  - '%MINICONDA_HOME%\Scripts\conda create -q -p C:\pythontest python=%PYTHON_VERSION% pytest numpy libpython'
```

Comment:
Do we still need `libpython`?

Reply:
Yes I think we do, adding libpython back in was one of the last things I had to do to get it to build. Don't ask me why though